hopp/tools/resource/wind_tools.py

```diff
@@ -80,8 +80,27 @@ def parse_resource_data(wind_resource):
     idx_ws = [ii for ii, field in enumerate(wind_resource.data['fields']) if field == 3]
     idx_wd = [ii for ii, field in enumerate(wind_resource.data['fields']) if field == 4]
     
-    # If there's only one hub-height, grab speed and direction data
-    if len(idx_ws)==1:
+    # If there's multiple hub-heights - average the data
+    if len(idx_ws) > 1:
+        hh1, hh2 = np.unique(wind_resource.data['heights'])
+        
+        if hh1 == wind_resource.hub_height_meters:
+            idx_ws1 = [i for i in idx_ws if wind_resource.data['heights'][i] == hh1][0]
+            idx_wd1 = [i for i in idx_wd if wind_resource.data['heights'][i] == hh1][0]
+            speeds = data[:, idx_ws1]
+            wind_dirs = data[:, idx_wd1]
+
+        elif hh2 == wind_resource.hub_height_meters:
+            idx_ws2 = [i for i in idx_ws if wind_resource.data['heights'][i] == hh2][0]
+            idx_wd2 = [i for i in idx_wd if wind_resource.data['heights'][i] == hh2][0]
+            speeds = data[:, idx_ws2]
+            wind_dirs = data[:, idx_wd2]
+        
+        else:
+            speeds = data[:, idx_ws].mean(axis=1)
+            wind_dirs = data[:, idx_wd].mean(axis=1)
+    else:
+        # If there's only one hub-height, grab speed and direction data
         speeds = data[:, idx_ws[0]]
         wind_dirs = data[:, idx_wd[0]]
         return speeds, wind_dirs
```

Comment:
These should now be in the top-most if-block. The `else` attached to that `if` can now be removed with the code inside that block dedented.